common/messages/messages_en.go

```diff
@@ -170,6 +170,62 @@ var messagesEn = map[string]util.Message{
 		RC:          500,
 		Action:      "Wait for file share deletion",
 	},
+	"ListSubnetsFailed": {
+		Code:        "ListSubnetsFailed",
+		Description: "Unable to fetch list of subnets.",
```

Comment:
I think description should be `Unable to fetch list of subnet.` same try to come-up with some meaningful name for Code as well

Reply:
We will check with Derek and update in next PR on error codes. Adding to pending PR https://github.ibm.com/alchemy-containers/armada-storage/issues/4583